code/game/objects/items/devices/uplink.dm

```diff
@@ -307,6 +299,7 @@ A list of items and costs is stored under the datum of every game mode, alongsid
 				contract["reward_other"] = select_query.item[6]
 
 				nanoui_data["contract"] = contract
+				log_debug("got the contractdata: [json_encode(contract)]")
```

Comment:
Please add a prefix to this debug line so we know what this message is coming from.

Maybe: `ContractUplink: `